src/app/[locale]/viewer/_components/ManualAasViewerInput.tsx

```diff
@@ -45,20 +45,28 @@ export function ManualAASViewerInput(props: { focus: boolean }) {
         try {
             setIsLoading(true);
 
-            const { registrySearchResult, aasId} = await getAasFromExternalServices(val);
-            const aas =
-                registrySearchResult != null
-                    ? registrySearchResult.registryAas
-                    : await repositoryClient.getAssetAdministrationShellById(encodeBase64(aasId));
-            
-            setAas(aas);
-            registrySearchResult?.registryAasData != null
-                ? setRegistryAasData({
-                    submodelDescriptors: registrySearchResult?.registryAasData?.submodelDescriptors,
-                    aasRegistryRepositoryOrigin: registrySearchResult?.registryAasData?.aasRegistryRepositoryOrigin })
-                : setRegistryAasData(null);
+            const aasIds = await handleAasDiscoverySearch(val);
+            if (aasIds && aasIds.length > 1) {
+                navigate.push(`/viewer/discovery?aasId=${val}`);
```

Comment:
Is `val` an Asset or AAS Id? the discovery only makes sense if it is an asset Id correct?